tests/test_interface_parser.py

```diff
@@ -199,16 +195,31 @@ def test_default_arguments(self):
         self.assertEqual(args[6].default, 3)
         # No default argument should set `default` to None
         self.assertIsNone(args[7].default)
-
         self.assertEqual(args[8].default, 3.1415)
 
+        args = ArgumentList.rule.parseString("""
+            gtsam::KeyFormatter kf = gtsam::DefaultKeyFormatter,
+            std::vector<size_t> v = std::vector<size_t>(),
+            std::vector<size_t> l = {},
```

Comment:
Ahhh good catch. Super glad we got you on board this project :smile:

Reply:
Also, I don't want to support lambdas as default arguments. They seem counterintuitive since one of the points of a lambda is to provide an easy one-off.

Reply:
K, tbh I mainly just wanted a test that used a comma inside square brackets and a lambda was the first language feature I could think of that used that in a literal expression 😂